classes/studentclass.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+# student subclass
+
+class Student < Person
```

Comment:
- [ ] Kindly remove the suffix class from the class files to ensure a proper class-file naming convention. This will prevent import errors. Kindly change the file `studentclass.rb` to `student.rb`